linux_install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# This is the install script for installing vs code onto a linux machine
+# Usage: sudo bash linux_install.sh 
+userid=`id -u`
+if [[ $userid -ne 0 ]]
+then
+   echo "Sorry, you aren't a root user"
+fi
+
+cmd=`mkdir "/usr/local/vscode/"`
+
+if [[ $cmd -eq 0 ]]; then
+	echo "folder creation successful"
+else
+	echo "something went wrong creating folder"
+fi
+
+cmd=`cp vscode.desktop /usr/share/applications`
+
+if [[ $cmd -eq 0 ]]; then
+	echo "successfully installed app shortcut"
+else
+	echo "something went wrong with installing app shortcut"
+fi
+
+cmd=`cp -r * /usr/local/vscode/`
+
+if [[ $cmd -eq 0 ]]; then
+	echo "successfully installed app"
+else
+	echo "something went wrong with installing app"
+fi
+
+echo 'export PATH=$PATH:/usr/local/vscode' >> ~/.bashrc
```

Comment:
NO. You install things into /usr/local, without tracking.
YOU WRITE THINGS INTO BASHRC!!! And apparently root's bashrc.
What's wrong with you?

This code isn't safe, it doesn't have `undo` command and it's tailored for one user only.